src/main/java/org/graylog/plugins/usagestatistics/UsageStatsOptOutState.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2015 Graylog, Inc. (hello@graylog.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.graylog.plugins.usagestatistics;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+@JsonAutoDetect
+@AutoValue
+public abstract class UsageStatsOptOutState {
+    @JsonProperty
```

Comment:
I would add the explicit property name here as well, otherwise we pretty much rely on the naming strategy of the given `ObjectMapper`.